packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DonutChartDemo/DonutSmallDemo.tsx

```diff
@@ -9,8 +9,8 @@ export class DonutSmallDemo extends React.Component {
 
   render() {
     return (
-      <div style={{width: '20%', paddingTop: '50px', paddingLeft: '50px'}}>
-        <div className="donut-chart-sm">
+      <div>
+        <div style={{height: '200px', width: '300px', paddingTop: '50px', paddingLeft: '50px'}}>
```

Comment:
This one appears too large because there is no legend. For this demo, the height and with should both be 150px.